applepay/utils.py

```diff
@@ -1,20 +1,38 @@
+"""
+The utils in this module are used to verify and parse a decoded
```

Comment:
Could you explain how a token is decoded in this context? My understanding from the documentation is that these tokens are plain-text JSON and not generally also base 64 encoded.